apps/minifront/src/state/swap/index.ts

```diff
@@ -15,6 +15,7 @@ import {
   InstantSwapSlice,
   createInstantSwapSlice,
   instantSwapSubmitButtonDisabledSelector,
+  isValidAmount,
```

Comment:
Please extract `isValidAmount` to `./helpers.ts`, so that slices aren't reaching into each other's code.